### ISSUE TITLE
Non-informative incorrect behaviour when using bad token (at least on windows x64)

### ISSUE THREAD

**Issue by eacp** [ipinfo/cli#57]
The cli says the login was successful whether the token is valid or not.

Steps to reproduce:

`ipinfo login`

Write any random string. The CLI does not validate it. It will say the token was configured.

`ipinfo 1.1.1.1`

The CLI informs the credentials and the token is incorrect, even though the `login` subcommand previously informed the token is correct.

**Comment by coderholic** [comment]
Thanks for flagging this. The login command should definitely confirm it's a valid token by making a requests.

**Comment by UmanShahzad** [comment]
Would we wanna check against the normal, core API, or introduce a new API for this just to check token validity? (with rate limits to prevent brute force token enumeration)

**Comment by eacp** [comment]
@UmanShahzad one option would be to just make a request with the input token and look for errors. Like checking the client's IP.

**Comment by eacp** [comment]
True. If that quota is really a concern, we could have tokens signed with a HMAC or could addapt the site to generate [JWTs](https://jwt.io/), wich are already signed. The CLI could contain the public key and the server could sign them with a private key using asymetric cryptography, like RSA or ECDSA. However, I imagine changing the entire process for generating tokens might be overkill if we just want to prevent **one** request being discounted from the quota.

**Comment by UmanShahzad** [comment]
That is definitely overkill. Introducing an API that returns "true" or "false" for a token's validity sounds easier.

I'll get on it next week.

**Comment by coderholic** [comment]
You can hit /me to check if the token is valid without using quota

**Comment by eacp** [comment]
I think a new API wouldn't even be neccesary. You could check a random IP address, or `myip` and check if there are any auth errors.

**Comment by UmanShahzad** [comment]
@coderholic Ah I was thinking we never properly publicized that API, but if you're okay with it then let's use that.

@eacp If the token was valid in that case, it would use up quota, going back to the original issue.

**Comment by eacp** [comment]
So, should I make a PR to use the /me api?

**Comment by UmanShahzad** [comment]
@eacp If you could that'd be great!

**Comment by eacp** [comment]
@UmanShahzad Done https://github.com/ipinfo/cli/pull/58
